docs/src/background/other_projects.rst

```diff
@@ -0,0 +1,360 @@
+.. Copyright 2023 CMakePP
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+.. http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+
+##############
+Other Projects
+##############
+
+.. warning::
+
+   The discussions of other projects provided on this page are largely based off
+   of first impressions gleaned from project websites and documentation. It is
+   possible that this page does not accurately reflect the state of the listed
+   projects. We are happy to update this page if discrepancies are brought to
+   our attention.
+
+.. note::
+
+   Subject to the above warning, note that each description was accurate as of
+   the time of writing. The CMaize team does not actively monitor other
+   projects and thus circumstances may have changed. If you are aware of any
+   of these descriptions no longer being accurate, feel free to open an issue
+   on CMaize's GitHub page (or better yet, a pull request) detailing the
+   necessary updates.
+
+This page discusses other projects which, in some shape or form, attempt
+to address the same needs as CMaize (see :ref:`statement_of_need`). As described
+in more detail on the :ref:`overview_of_cmaizes_design` page, we feel the ideal
+solution should:
+
+1. :ref:`cmake_based_build_system`
+2. :ref:`cmake_based_workflows`
+3. :ref:`minimize_redundancy`
+4. :ref:`target_support`
+5. :ref:`package_manager_support`
+6. :ref:`object_oriented`
+7. :ref:`recursive`
+8. Be actively supported.
+
+The following table summarizes how well each existing project addresses these
+design constraints (column numbers correspond to the above list):
+
+.. |y| replace:: ✅
```

Comment:
The symbols you are using in the table for `|y|`, `|n|`, and `|?|` don't seem to be rendering properly in the GitHub text editor. It also doesn't appear for me in VSCodium or in my browser when I build the docs. We may want to consider using different symbols here.

Reply:
Hmm. They render just fine for me on GitHub and in the built documentation. Not sure how else to put emojis into the documentation aside from copy/paste. If they still don't show up for you in the online docs, after CI/CD builds it, I'll revisit them.